gradle.properties

```diff
@@ -0,0 +1,5 @@
+org.gradle.daemon=true
```

Comment:
Why do you think this is required to be in master?

Reply:
Which part? Gradle daemon or gradle.properties?